docs/thirdparty.rst

```diff
@@ -37,7 +37,20 @@ Third Party Applications
 .. content list start
 
 
-.. replace this line with the first entry
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
```

Comment:
Could you please use a URL that points to a tag or commit id? Using the master branch reference might cause future issues... 😅 
A better alternative that you could use (ProTip) is pasting the image to a new issue thread *on your repo* (!!!) and using this newly created link. You don't even have to submit the issue and the image will be hosted by Github outside of the git repo.